### ISSUE TITLE
（超時空要塞マクロス）OpenGL and Vulkan mode screen flashes

### ISSUE THREAD

**Issue by Nova-0** [Vita3K/compatibility#905]
# App summary
- App name: OpenGL and Vulkan mode screen flashes
- App serial: PCSG00947
- App version: 1.00

# Vita3K summary
- Version: v0.1.6
- Build number: 2825- Commit hash: https://github.com/vita3k/vita3k/commit/[commit] 
- CPU backend: 
                          

# Test environment summary
- Tested by: Nova-0
- OS: Windows 10
- CPU: Intel
- GPU:NVIDIA
- RAM: 16GB

# Issues
OpenGL and Vulkan mode screen flashes

# Screenshots
![{9DU(QRMVF5RYCNY)5PX{6D](https://user-images.githubusercontent.com/113018174/188841148-4a3e4a0d-1c20-430d-8a94-a0599614918f.png)

# Log
[vita3k.log](https://github.com/Vita3K/compatibility/files/9504608/vita3k.log)
# Recommended labels
- playable
- screen flashing

**Comment by PaladinChalybs** [comment]
Tested, still not playable on my PC. Screen is mostly black with random flashes showing game.
 i5-11300H @ 3.10GHz, FX516PC, 16Gb DDR4, GeForce RTX 3050 4Gb GDDR6, Windows 10 Home.
Log attached.
[vita3k.log](https://github.com/Vita3K/compatibility/files/9995278/vita3k.log)

**Comment by KronicKiller** [comment]
same gota working hub but none of the level
ocational flashing of the level

**Comment by KronicKiller** [comment]
https://www.youtube.com/watch?v=KDf4N1vLlw0
mine does exactly that
3900x with 3090 windows 11

**Comment by Zazazasu** [comment]
same here

**Comment by Zinogre331** [comment]
yeah its not playable the gameplay is just a flashing screen black and white. the UI works perfectly fine tho. Hoping this will get fix soon ty.

**Comment by McFly5050** [comment]
Same issues above as well. Just black screen.  Would love to see this game working.

**Comment by cityracoon** [comment]
maybe this is possible 
I played this game today. it is possible
![2023-10-29 23 23 51](https://github.com/Vita3K/compatibility/assets/19235284/f032be0b-de50-4c13-ae8b-83e6d9892053)

**Comment by BriaN890501** [comment]
Can confirm playable(on steam deck) but there's still some issue
OpenGL cause UI glitches / Some effect (missile trail, engine flare, bullet) missing when using vulkan

**Comment by l2radamanthys** [comment]
Regular performance, some rendering issues on white backgrounds with OpenGL. With Vulkan, rendering is better but tends to crash. Ammunition and missile trails are not visible (they are visible in some parts with OpenGL). The game UI works well, but it's cumbersome as playing a single mission may lead to a crash or hang on the next attempt.

SO: Windows 10
RTX 3080 TI
32 Gb RAM

**Comment by msolehab** [comment]
Same issue, after a single mission, next mission will cause the game to crash. On the other hand I have issue where some mech just have blinding lights, even turning off "Disable Surface Sync" won't help

**Comment by Haruto96** [comment]
Tested on Samsung Galaxy M54
Playable with a bit of slowdown but the artifacts really annoying 
![Screenshot_20241127_214350_Vita3K](https://github.com/user-attachments/assets/e7736651-3971-4d8c-8815-1c8e60545c9f)
![Screenshot_20241127_213245_Vita3K](https://github.com/user-attachments/assets/da524805-0712-40d6-900f-ca75efb14085)

**Comment by quyenphong1243-wq** [comment]
March 5 update: nothing new, the old issues are still there

**Comment by PaladinChalybs** [comment]
Playable. No visual glitches save for minor slowdowns. 0.2.1. - 3858-7b21f596

Vulkan. Same hardware. 

https://github.com/Vita3K/Vita3K-builds/releases/tag/3858

Updates disabled, just in case.

<img width="1869" height="1022" alt="Image" src="https://github.com/user-attachments/assets/85251d14-ca15-4085-bbf1-9f1ffa1ed85b" />
